trade-finance-manager-ui/component-tests/utilisation-reports/add-to-an-existing-payment.component-test.ts

```diff
@@ -93,4 +93,24 @@ describe(page, () => {
     wrapper.expectElement('[data-cy="continue-button"]').toExist();
     wrapper.expectText('[data-cy="continue-button"]').toRead('Continue');
   });
+
+  it('should render the cancel link', () => {
+    // Arrange
+    const addToAnExistingPaymentViewModel = anAddToAnExistingPaymentViewModel();
+    const wrapper = render(addToAnExistingPaymentViewModel);
+
+    // Assert
+    wrapper.expectElement('[data-cy="cancel-link"]').toExist();
+    wrapper.expectText('[data-cy="cancel-link"]').toRead('Cancel');
+  });
+
+  it('should render the back link', () => {
+    // Arrange
+    const addToAnExistingPaymentViewModel = anAddToAnExistingPaymentViewModel();
+    const wrapper = render(addToAnExistingPaymentViewModel);
+
+    // Assert
+    wrapper.expectElement('[data-cy="back-link"]').toExist();
+    wrapper.expectText('[data-cy="back-link"]').toRead('Back');
+  });
```

Comment:
Could we assert that these links are set to the url given by the view model? Unless there is a reason we can't use the `expectLink(...).toLinkTo(...)` assertions that should be the nicest way to do it.